include/smack/Naming.h

```diff
@@ -90,6 +91,9 @@ class Naming {
   static const std::string MEMORY_SAFETY_FUNCTION;
   static const std::string MEMORY_LEAK_FUNCTION;
 
+  enum RMode { RNE, RNA, RTP, RTN, RTZ };
```

Comment:
This is used like an enum class in Naming.cpp. Please update this declaration accordingly.